README.md

```diff
@@ -108,6 +109,20 @@ The HOT Export pipeline depends on a number of third-party tools.
 
 <code>$ sudo apt-get install default-jre zip unzip</code>
 
+Install Qt 5.5:
+
+<code>$ sudo add-apt-repository ppa:beineri/opt-qt55-trusty </code>
+
+<code>$ sudo apt-get update </code>
+
+<code>$ sudo apt-get install qt55base </code>
+
+<code>$ source /opt/qt55/bin/qt55-env.sh </code>
```

Comment:
https://github.com/hotosm/osm-export-tool2/issues/42 . Referring to the comments made by @Zverik . For the generator tool to work we need it installed. Referred form :- https://github.com/mapsme/omim/blob/master/docs/INSTALL.md